homeassistant/components/reolink/__init__.py

```diff
@@ -234,3 +219,17 @@ def cleanup_disconnected_cams(
 
         # clean device registry and associated entities
         device_reg.async_remove_device(device.id)
+
+
+def migrate_entity_ids(
+    hass: HomeAssistant, config_entry_id: str, host: ReolinkHost
+) -> None:
+    """Migrate entity IDs if needed."""
+    entity_reg = er.async_get(hass)
+    entities = er.async_entries_for_config_entry(entity_reg, config_entry_id)
+    for entity in entities:
+        # Can be remove in HA 2025.1.0
```

Comment:
removed

Reply:
oops, thanks for pointing out.
I fix the typo in this PR https://github.com/home-assistant/core/pull/119241
(It is so small I did not want to create a dedicated PR for it).